contracts/FraudProof.sol

```diff
@@ -35,69 +34,13 @@ contract FraudProofHelpers is FraudProofSetup {
         }
 
         // check from leaf has enough balance
-        if (_from_account.balance < amount.add(fee)) {
+        if (fromState.balance < amount.add(fee)) {
             // invalid state transition
-            // needs to be slashed because the account doesnt have enough balance
+            // needs to be slashed because the state doesnt have enough balance
             // for the transfer
             return Types.ErrorCode.NotEnoughTokenBalance;
         }
 
         return Types.ErrorCode.NoError;
     }
-
-    function RemoveTokensFromAccount(
```

Comment:
We removed these functions: RemoveTokensFromAccount, UpdateBalanceInAccount, AddTokensToAccount, and BalanceFromAccount.